packages/eslint-plugin-nimble/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Could you add a couple-sentence README? Basically just your existing description below plus mention that it is built on the NI style guide (and link to that repo).